src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1382,15 +1407,13 @@ internal enum ErrorCode
         ERR_BadIteratorLocalType = 8941,
         ERR_BadAsyncLocalType = 8942,
         ERR_RefReturningCallAndAwait = 8943,
+
+        #endregion diagnostics for ref locals and ref returns introduced in C# 7
+
+        #region diagnostics introduced for original/replace in C# 7
         ERR_NoOriginalMember = 8944,
         ERR_DuplicateReplace = 8945,
         ERR_PartialReplace = 8946,
-        ERR_TupleTooFewElements = 8200,
-        ERR_TupleReservedMemberName = 8201,
-        ERR_TupleReservedMemberNameAnyPosition = 8202,
-        ERR_TupleDuplicateMemberName = 8203,
-        ERR_TupleExplicitNamesOnAllMembersOrNone = 8204,
-
-        ERR_PredefinedTypeMemberNotFoundInAssembly = 8205,
+        #endregion diagnostics introduced for original/replace in C# 7
```

Comment:
I am not sure if introducing these regions is useful and should be done. As far as I remember we never attributed particular ranges of error numbers to particular versions or features. And it was always OK to grab any number available, assuming the number wasn't used by a shipped product in the past, from any gap one can find. If you are trying to introduce a new policy, I think it should be discussed first. Otherwise, it feels like these regions will only confuse people editing/reading the code.